test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -1258,15 +1258,15 @@ public IOcelotLogger CreateLogger<T>()
                 if (_logger == null)
                 {
                     _logger = new Mock<IOcelotLogger>();
-                    _logger.Setup(x => x.LogWarning(It.IsAny<string>())).Verifiable();
+                    _logger.Setup(x => x.LogWarning(It.IsAny<Func<string>>())).Verifiable();
```

Comment:
You had to change a lot of test classes because you've removed old signature method.

Reply:
yes, I'm a refactoring menace